site/.vitepress/config.ts

```diff
@@ -122,9 +110,33 @@ export default defineConfig({
               { text: "Contributing Your Signer", link: "/contributing" },
             ],
           },
-          { text: "Sponsoring Gas", link: "/sponsoring-gas" },
-          { text: "Batching Transactions", link: "/batching-transactions" },
-          { text: "Transferring Ownership", link: "/transferring-ownership" },
+        ],
+      },
+      {
+        text: "Guides",
+        base: "/guides",
+        items: [
+          {
+            text: "How to Send a User Operation",
+            link: "/send-user-operation",
+          },
+          {
+            text: "How to Sponsor Gas for a User Operation",
+            link: "/sponsoring-gas",
+          },
+          { text: "How to Batch Transactions", link: "/batching-transactions" },
+          {
+            text: "How to Transfer Ownership of a Smart Account",
+            link: "/transferring-ownership",
+          },
+          {
+            text: "How to Fetch a Smart Account's NFTs",
+            link: "/enhanced-apis-nft",
+          },
+          {
+            text: "How to Fetch a Smart Account's ERC-20 Tokens",
+            link: "/enhanced-apis-token",
+          },
```

Comment:
the texts are too long here and will show cut off on the sidebar. Can we make the site sidebar have maxWidth and wrap long texts on the sidebar?

Reply:
They wrap! See screen recording on slack post for pr

Reply:
yup they do :) sorry!